ci/Makefile

```diff
@@ -59,6 +59,8 @@ update_jenkins_jobs:
 
 .PHONY: update_pkgs
 update_pkgs:
+	sudo zypper in -y go1.13
```

Comment:
I think this kind of changes should be coordinated with the Z-squad.  

cc @jordimassaguerpla